src/lib/journey/journey.store.ts

```diff
@@ -96,6 +96,11 @@ export const journeyStore: Writable<JourneyStoreValue> = writable({
   successful: false,
   response: null,
   recaptchaAction: null,
+  pingProtect: {
+    envId: '',
+    consoleLogEnabled: false,
+    pauseBehavioralData: false,
+  },
```

Comment:
Is there a reason we are just choosing these three properties?

Reply:
no i think we can remove them all except envId